Makefile

```diff
@@ -72,15 +81,15 @@ package: installables
 	   	"$(OUTPUT_PACKAGE)"
 
 prefix_install: installables
-	$(MKDIR) "$(BINARIES_FOLDER)"
-	$(CP) -f "$(CARTHAGE_EXECUTABLE)" "$(BINARIES_FOLDER)/"
+	/bin/zsh --no-globalrcs --no-rcs -c 'set -x; mkdir -p "$${BINARIES_FOLDER:?}"'
+	$(CP) -v -f "$(CARTHAGE_EXECUTABLE)" "$${BINARIES_FOLDER:?}"
 
 install: installables
-	if [ ! -d "$(BINARIES_FOLDER)" ]; then $(SUDO) $(MKDIR) "$(BINARIES_FOLDER)"; fi
-	$(SUDO) $(CP) -f "$(CARTHAGE_EXECUTABLE)" "$(BINARIES_FOLDER)"
+	if [ ! -d "$${BINARIES_FOLDER:?}" ]; then $(SUDO) mkdir -p "$${BINARIES_FOLDER:?}"; fi
+	$(SUDO) $(CP) -v -f "$(CARTHAGE_EXECUTABLE)" "$${BINARIES_FOLDER:?}"
 
 uninstall:
-	$(RM) "$(BINARIES_FOLDER)/carthage"
+	/bin/zsh --no-globalrcs --no-rcs -c 'set -x; rm -f -v "$${BINARIES_FOLDER:?}"'
```

Comment:
Just fixed up (well before landing this PR) in https://github.com/Carthage/Carthage/pull/3352/commits/a9908175afa3ff0540cedb8e00d072d4c39933a9.